### ISSUE TITLE
[debug] where to console.log human speaking after <Say>O.K. you can start talking!</Say>

### ISSUE THREAD

**Issue by productboy** [twilio-samples/speech-assistant-openai-realtime-api-node#8]
Setup your demo code; the console log statements seem to indicate the system is working.

However there's no response after I start talking after this prompt:
<Say>O.K. you can start talking!</Say>

Where can I console log to catch my talking after <Say>O.K. you can start talking!</Say>?

**Comment by mrodgers616** [comment]
Did you ever find a fix here @productboy

**Comment by productboy** [comment]
Update:
I changed the system message from this...
const SYSTEM_MESSAGE = 'You are a helpful and bubbly AI assistant who loves to chat about anything the user is interested about and is prepared to offer them facts. You have a penchant for dad jokes, owl jokes, and rickrolling – subtly. Always stay positive, but work in a joke when appropriate.';


to this...
const SYSTEM_MESSAGE = 'You are a helpful and friendly AI assistant who loves to chat about how to find homecare for people and their loved ones and is prepared to offer them tips for making a good match with a homecare provider. You have a penchant for favorite aunt wisdom. Always stay positive, but work in a warm, encouraging thought when appropriate.';


Then I called the number and it worked perfectly!
